api/Navigator.json

```diff
@@ -1404,10 +1404,12 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": "preview",
+              "version_added": "120",
               "notes": "Opt-in to GPC by setting the preference <code>privacy.globalprivacycontrol.enabled</code> to <code>true</code>."
```

Comment:
@Elchi3 Finally confirmed that there is a proper UI for enabling GPC. This is a setting in  `about:preferences#privacy` ( currently on nightly since this isn't released yet):

![image](https://github.com/mdn/browser-compat-data/assets/5368500/9209b244-c128-4152-94f0-df6eeab0bab4)

This maps to the preference flag so you can set either. So suggesting I change the text - is it worth mentioning both?

```suggestion
              "notes": "Opt-in to GPC using the Website Privacy Preference setting (<code>about:preferences#privacy</code>) checkbox 'Tell websites not to sell or share my data', or by setting the preference <code>privacy.globalprivacycontrol.enabled</code> to <code>true</code>."
```

Note: 
- I'll roll out what we agree on to the HTTP header too.
- The spec does not outline how the feature should be enabled, so it is is a kind of compatibility issue in that it will be different in every browser. That's why I am documenting it here.

Reply:
Sounds good to me!